src/main/resources/api.yaml

```diff
@@ -329,6 +379,12 @@ components:
         description:
           type: "string"
           maxLength: 500
+        blockUntil:
+          type: "string"
+          format: "date-time"
+        nthTrial:
+          type: "integer"
+          minimum: 0
```

Comment:
I think we don't need to expose these information to the API?

Reply:
Testing code. Now deleted.